### ISSUE TITLE
have systems code page get autogenerated

### ISSUE THREAD

**Issue by egonw** [bridgedb/bridgedb.github.io#5]
The Markdown for https://bridgedb.github.io/pages/system-codes.html should be autogenerated.

**Comment by egonw** [comment]
And should include the prefix too...

**Comment by DeniseSl22** [comment]
Agreed, now we also (still) list UniProt Trembl vs Swissprot, which has been merged in one (UniProt):
![image](https://user-images.githubusercontent.com/26277832/213154515-3d86a49f-8d9d-47c4-b7e2-e91a33099850.png)

**Comment by egonw** [comment]
It has not been merged yet. It's complicated because several tools use string matching to look up a data source. So, the system code `S` needs to be linked to the full name `Uniprot-TrEMBL` and `UniProt`. I am not sure that works and have not had time to test that with BridgeDb yet. Maybe at the upcoming hackathon?

**Comment by Chris-Evelo** [comment]
There is a real content difference between UniProt and trEMBL even though they are merged into UniProt. trEMBL is translated EMBL and thus in principle is about hypothetical proteins. UniProt is about biochemically known proteins. So during analysis, you typically can want to figure out to which set a protein belongs. For human, that might no longer be relevant assuming that not only have most real proteins been found but also that all really hypothetical and probably non-existent proteins have been annotated as such. But I would think it is still relevant for other species.

**Comment by egonw** [comment]
> There is a real content difference between UniProt and trEMBL even though they are merged into UniProt. trEMBL is translated EMBL and thus in principle is about hypothetical proteins. UniProt is about biochemically known proteins. So during analysis, you typically can want to figure out to which set a protein belongs. For human, that might no longer be relevant assuming that not only have most real proteins been found but also that all really hypothetical and probably non-existent proteins have been annotated as such. But I would think it is still relevant for other species.

Yes, I know. This issue is not about that. UniProt is now an integrated resource and we can use a federate approach to see if a UniProt entry is in TrEBML or SwissProt.
